src/components/Tooltip/Tooltip.js

```diff
@@ -103,7 +117,12 @@ class Tooltip extends PureComponent {
           onRootClose={this.handleHideToolTip}
           style={overlayStyle}
         >
-          <InnerToolTip size={size} style={style} arrowStyle={arrowStyle} snacksStyle={snacksStyle}>
+          <InnerToolTip
+            size={size}
+            customStyle={customStyle || style}
```

Comment:
should we drop a comment here to remove the `|| style` once we remove support for `style`?

Reply:
`style` will be undefined here once we remove it from props, which should enforce its removal here. I'm fine to add @todos if you'd like, just didn't want to create too much mess in Snacks. Thoughts?

Reply:
Sounds good 👍